src/types/configData.go

```diff
@@ -111,11 +111,17 @@ type ConfigPublisher struct {
 }
 
 type ConfigPurgeOldDBEntries struct {
-	Enable              bool     `json:"enable,omitempty" bson:"enable,omiempty"`
+	Enable              bool     `json:"enable,omitempty" bson:"enable,omitempty"`
 	CronJobTimeInterval string   `json:"cronjob_time_interval,omitempty" bson:"cronjob_time_interval,omitempty"`
 	DBName              []string `json:"db_name,omitempty" bson:"db_name,omitempty"`
 }
 
+type ConfigRecommendPolicy struct {
+	OperationMode           int    `json:"operation_mode,omitempty" bson:"operation_mode,omitempty"`
+	CronJobTimeInterval     string `json:"cronjob_time_interval,omitempty" bson:"cronjob_time_interval,omitempty"`
+	OneTimeJobTimeSelection string `json:"one_time_job_time_selection,omitempty" bson:"one_time_job_time_selection,omitempty"`
```

Comment:
The configuration for the same is not provided in conf/local-file.conf

Reply:
it is @seswarrajan 

```
# Recommended policies configuration
recommend:
  operation-mode: 1                       # 1: cronjob | 2: one-time-job
  cron-job-time-interval: "1h0m00s"       # format: XhYmZs
```